src/main/java/entity/Armor.java

```diff
@@ -0,0 +1,16 @@
+package entity;
+
+public class Armor extends Equipment {
+
+    public Armor(String name, int num, int dmgReduced){
```

Comment:
Reminder: if we remove 'num' attribute from Item class, you can remove it as an argument here.